pkg/armhelpers/azureclient.go

```diff
@@ -211,7 +217,6 @@ func NewAzureClientWithClientSecretExternalTenant(env azure.Environment, subscri
 	if err != nil {
 		return nil, err
 	}
-	graphSpt.Refresh()
```

Comment:
This was returning a consistent error (once we started checking it). Need to investigate more.

Reply:
I changed this and similar calls to `log.Error(err)` so as not to change existing control flow.